src/components/panel/panel.js

```diff
@@ -306,6 +309,8 @@ angular
  * Adds a class to the panel. DO NOT use this to hide/show the panel.
  *
  * @param {string} newClass Class to be added.
+ * @param {boolean} toElement Whether or not to add the class to the panel
```

Comment:
@ThomasBurleson I'm fine with whatever documentation style you want in ngMaterial. The only thing I really feel strongly about is that every method have some form of docs so that anyone reading it can understand what that method does. I've found this really helpful for someone adding to the component after it's created.

The public vs private was so that we didn't use ngdoc on every method (therefore writing every method on the docs site, even private ones). I thought it made the public API more clear to have it at the beginning, but I understand that makes it harder to maintain. I've seen the list everything up front pattern in a few other components, like mdDialog, but I'm open to new suggestions about documentation.